akvo/rsr/static/scripts-src/my-results/selectors.js

```diff
@@ -120,7 +120,8 @@ export const getPeriodsActualValue = createSelector(
                     return acc;
                 }, 0
             );
-            return {...acc, [periodId]: actualValue};
+          const periodActualValue = (parseFloat(periodObjects[periodId].actual_value)||0);
```

Comment:
A comment here describing what's going on would be good I think. It's kinda non obvious...